packages/expo-dev-launcher/ios/EXDevLauncherURLHelper.swift

```diff
@@ -52,6 +53,18 @@ public class EXDevLauncherURLHelper: NSObject {
     return hasUrlQueryParam
   }
 
+  @objc
+  public static func disableOnboardingPopupIfNeeded(_ url: URL) {
+    let components = URLComponents.init(url: url, resolvingAgainstBaseURL: false)
+
+    for queryItem in components?.queryItems ?? [] {
+      if queryItem.name == "disableOnboarding" && queryItem.value != nil {
+        DevMenuPreferences.isOnboardingFinished = true
+        break
+      }
+    }
```

Comment:
```suggestion
    if components?.queryItems?.contains(where: { $0.name == "disableOnboarding" && $0.value != nil }) {
        DevMenuPreferences.isOnboardingFinished = true
    }
```

Reply:
Also, I think we should check the exact value of the `disableOnboarding` parameter and compere it to 1.

Reply:
Made the change to this syntax (also did this for the other method above, and made it check for the value 1. Ran `swiftlint --fix` on this file as well.